XamlControlsGallery/ControlPages/TabViewPage.xaml.cs

```diff
@@ -210,10 +210,32 @@ private void TabWidthBehaviorComboBox_SelectionChanged(object sender, SelectionC
                 case "SizeToContent":
                     widthMode = TabViewWidthMode.SizeToContent;
                     break;
+                case "Compact":
+                    widthMode = TabViewWidthMode.Compact;
+                    break;
             }
             TabView3.TabWidthMode = widthMode;
         }
 
+        private void TabCloseButtonOverlayModeComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            string widthModeString = (e.AddedItems[0] as ComboBoxItem).Content.ToString();
+            TabViewCloseButtonOverlayMode overlayMode = TabViewCloseButtonOverlayMode.Auto;
+            switch (widthModeString)
+            {
+                case "Auto":
+                    overlayMode = TabViewCloseButtonOverlayMode.Auto;
+                    break;
+                case "OnHover":
+                    overlayMode = TabViewCloseButtonOverlayMode.OnHover;
+                    break;
+                case "Always":
+                    overlayMode = TabViewCloseButtonOverlayMode.Always;
+                    break;
+            }
+            TabView4.CloseButtonOverlayMode = overlayMode;
```

Comment:
Oops, maybe the name is required? Why doesn't this crash?

Reply:
I've updated both names at the same time so they still match up.